test/imageVoterContract.js

```diff
@@ -0,0 +1,38 @@
+const ImageVoterContract = artifacts.require("../ImageVoterContract.sol");
+
+contract("ImageVoterContract", accounts => {
+  it("should store the images metadata", async () => {
+    const contract = await ImageVoterContract.deployed();
+
+    const idOne = web3.utils.fromAscii("one")
```

Comment:
Curious where `web3` is from? Doesn't seem to be imported.

Reply:
@bjing Truffle implicitly imports it [DOCS](https://www.trufflesuite.com/docs/truffle/testing/writing-tests-in-javascript#using-web3)